README.md

```diff
@@ -17,7 +17,6 @@ If you want to contribute, please read [this](CONTRIBUTING.md).
     - [Observability](#observability)
     - [Operating systems](#operating-systems)
     - [Productivity](#productivity)
-    - [Security tools](#security-tools)
```

Comment:
Why was this deleted?

Reply:
My bad! I made these changes in the GitHub web interface and the find command they used must have dome something. I'll fix these :)